src/Features/CSharp/Portable/CodeFixes/Nullable/CSharpDeclareAsNullableCodeFixProvider.cs

```diff
@@ -80,8 +80,7 @@ private static TypeSyntax TryGetDeclarationTypeToFix(SyntaxNode node)
 
             if (node.IsParentKind(SyntaxKind.ReturnStatement))
             {
-                var containingMember = node.GetAncestors().FirstOrDefault(a => a.IsKind(SyntaxKind.MethodDeclaration, SyntaxKind.PropertyDeclaration,
-                    SyntaxKind.ParenthesizedLambdaExpression, SyntaxKind.SimpleLambdaExpression, SyntaxKind.LocalFunctionStatement, SyntaxKind.AnonymousMethodExpression));
+                var containingMember = node.GetAncestors().FirstOrDefault(a => a.IsReturnableConstruct());
```

Comment:
Does that work properly for accessors/properties?

Reply:
Hum, good catch. A test did regress. I'll adjust.